apps-rendering/src/mainMedia.ts

```diff
@@ -0,0 +1,15 @@
+// ----- Imports ----- //
```

Comment:
Should we have a `/types` direcotry in src for files only containing types? I think we've got a few files like this that can be tucked away neatly in there

Reply:
I think I'd prefer not to consider types as a separate "thing" to the rest of the code 🤔. With TypeScript we can treat types as no different to any other feature of the language. In my head I can see creating a directory for files that just contain types as analogous to creating a directory called "functions" where we keep files that only have functions in them, or one called "variables" where we keep files that only have variables in them?

I'm aware that what you're suggesting is maybe more geared toward tidiness than treating types as inherently different, but I worry it may look to future developers that "this is the place where we keep our types, and over here is the place where we keep the rest of our code"?

A pattern that we've tried to follow in AR is to organise modules around a type that models a particular feature and the functions that operate on that type. For example, `liveBlock.ts`, `football.ts`, `embed.ts` and so on. I think this may be a common approach found in other typed languages, and could be considered analogous to organising files around classes in Java/Scala. In some cases a type might not have any functions living alongside it at the moment, but I think that's ok?

Let me know what you think.